clients/go-core/gocore.sh

```diff
@@ -116,15 +123,15 @@ fi
 
 # Configure any mining operation
 if [ "$HIVE_MINER" != "" ] && [ "$HIVE_NODETYPE" != "light" ]; then
-    FLAGS="$FLAGS --mine --miner.coinbase $HIVE_MINER"
+    FLAGS="$FLAGS --mine --miner.corebase $HIVE_MINER --miner.threads=99"
 fi
 if [ "$HIVE_MINER_EXTRA" != "" ]; then
     FLAGS="$FLAGS --miner.extradata $HIVE_MINER_EXTRA"
 fi
 
-# Configure LES.
+# # Configure LES.
 if [ "$HIVE_LES_SERVER" == "1" ]; then
-  FLAGS="$FLAGS --light.serve 50 --light.nosyncserve"
+  FLAGS="$FLAGS --light.serve 80 --bttp "
```

Comment:
http?

Reply:
no, it's `--bttp` flag. 
`  --bttp                              Broadcasting blocks at first to trusted peers and then to all connected peers`

Reply:
no, it's `--bttp` "blocks-to-trusted-peers" flag. 
`  --bttp                              Broadcasting blocks at first to trusted peers and then to all connected peers`

Reply:
no, it's `--bttp` "blocks-to-trusted-peers" flag. 
`  --bttp                              Broadcasting blocks at first to trusted peers and then to all connected peers`